packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -580,6 +580,64 @@ class FadeTransition extends SingleChildRenderObjectWidget {
   }
 }
 
+/// Animates the opacity of a sliver widget.
```

Comment:
This might need more documentation, also some example code will be useful as well

Reply:
Done! :)